### PR TITLE
Phpcs fixes

### PR DESCRIPTION
**Pull request by trepmal**
General sprucing up.

 - escaping
 - formatting
 - better isset checks
 - whitespace trimming

### CODE REVIEW

**Comment by trepmal** [comment]
No null coalescing operator if we want to support PHP<7
https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op

**Comment by trepmal** [comment]
Adding license info in here just to get ahead of merge conflict.

**Comment by trepmal** [comment]
PHP 5 tests removed in #8

File: safe-report-comments.php [hunk 1]
```diff
@@ -261,132 +275,149 @@ public function mark_comment_moderated( $comment ) {
...
 						return true;
 					}
 				}
-			} 
-			
-			
+			}
+
+			// phpcs:ignore WordPressVIPMinimum.Variables.ServerVariables.UserControlledHeaders
+			$remote_addr = filter_var( isset( $_SERVER['REMOTE_ADDR'] ) ? $_SERVER['REMOTE_ADDR'] : '', 'FILTER_VALIDATE_IP' );
```
Comment (GaryJones) [inline safe-report-comments.php:295]: Consider changing to use `filter_input()`, which should then move away from needing the PHPCS ignore as well.

File: safe-report-comments.php [hunk 2]
```diff
@@ -261,132 +275,149 @@ public function mark_comment_moderated( $comment ) {
...
 					@setcookie( $this->_storagecookie, $cookie, time()+$this->cookie_lifetime, COOKIEPATH, COOKIE_DOMAIN );
-					if ( SITECOOKIEPATH != COOKIEPATH )
+					if ( SITECOOKIEPATH != COOKIEPATH ) {
 						@setcookie( $this->_storagecookie, $cookie, time()+$this->cookie_lifetime, SITECOOKIEPATH, COOKIE_DOMAIN);
+					}
 				}
 			}
+
+			// phpcs:ignore WordPressVIPMinimum.Variables.ServerVariables.UserControlledHeaders
+			$remote_addr = filter_var( isset( $_SERVER['REMOTE_ADDR'] ) ? $_SERVER['REMOTE_ADDR'] : '', 'FILTER_VALIDATE_IP' );
```
Comment (GaryJones) [inline safe-report-comments.php:343]: As above.

File: safe-report-comments.php [hunk 3]
```diff
@@ -261,132 +275,149 @@ public function mark_comment_moderated( $comment ) {
...
-		
-		/* 
+
+		/*
 		 * Ajax callback to flag/report a comment
 		 */
-		public function flag_comment() {		
-			if ( (int) $_REQUEST[ 'comment_id' ] != $_REQUEST[ 'comment_id' ] || empty( $_REQUEST[ 'comment_id' ] ) )
+		public function flag_comment() {
+			if ( ! isset( $_REQUEST[ 'comment_id' ] ) || empty( $_REQUEST[ 'comment_id' ] ) || (int) $_REQUEST[ 'comment_id' ] != $_REQUEST[ 'comment_id' ] ) {
```
Comment (GaryJones) [inline safe-report-comments.php:396]: No need for the `isset()` check if it's followed by `empty()` - `empty()` won't give any notice or warning if it's not set.

**Review by GaryJones — CHANGES_REQUESTED** [review]

**Review by GaryJones — APPROVED** [review]

**Pull request merged**